content/en/docs/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -4,96 +4,103 @@ description: Upgrading an ambient mode installation with Helm.
 weight: 5
 aliases:
   - /docs/ops/ambient/upgrade/helm-upgrade
-  - /latest/docs/ops/ambient/upgrade/helm-upgrade  
+  - /latest/docs/ops/ambient/upgrade/helm-upgrade
 owner: istio/wg-environments-maintainers
 test: yes
 status: Experimental
 ---
 
 Follow this guide to upgrade and configure an ambient mode installation using
-[Helm](https://helm.sh/docs/). This guide assumes you have already performed an [ambient mode installation with Helm](/docs/ambient/install/helm-installation/) with a previous minor or patch version of Istio.
+[Helm](https://helm.sh/docs/). This guide assumes you have already performed an [ambient mode installation with Helm](/docs/ambient/install/helm-installation/) with a previous version of Istio.
 
 {{< warning >}}
-In contrast to sidecar mode, ambient mode supports moving application pods to an upgraded data plane without a mandatory restart or reschedule of running application pods. However, upgrading the data plane **will** briefly disrupt all workload traffic on the upgraded node, and ambient mode does not currently support canary upgrades of the data plane.
+In contrast to sidecar mode, ambient mode supports moving application pods to an upgraded data plane without a mandatory restart or reschedule of running application pods. However, upgrading the ztunnel **will** briefly disrupt all workload traffic on the upgraded node, and ambient mode does not currently support canary upgrades of the ztunnel.
 
 Node cordoning and blue/green node pools are recommended to control blast radius of application pod traffic disruption during production upgrades. See your Kubernetes provider documentation for details.
 {{< /warning >}}
 
+## Understanding Ambient Upgrades
+
+All Istio Upgrades involve upgrading the control plane, data plane, and Istio CRDs. Because the Ambient Data Plane is separated into [two components](/docs/ambient/architecture/data-plane), the ztunnel and waypoints, Ambient Upgrades involve separate steps for these components. Upgrading the control plane and CRDs will be covered here in brief, but is essentially identical to the process for these components in sidecar mode, which is covered extensively [here](docs/setup/upgrade/canary/).
+
+Like sidecar mode, ambient mode upgrades make use of Tags and Revisions to allow fine-grained control over gateway upgrades, including waypoints, with simple controls for rolling back at any point. However, unlike sidecar mode, the Ambient ztunnel runs as a DaemonSet, or per-node proxy, meaning that ztunnel upgrades affect, at minimum, one entire node at a time. For this reason, we recommend using Node cordoning and draining before upgrading the ztunnel component for a given node. For the sake of simplicity, this document will demonstrate in-place upgrades of the ztunnel, which are less safe, and may involve some downtime.
```

Comment:
I really don't think we should imply that you should only use tags and revisions to upgrade, or even mention that first.

We support _2_ upgrade modes - inplace of all components (simple, fewer blast radius controls), tag + rev (complex, requires more operational planning, not even required if you A/B clusters for instance)

I think we have to document both, and start with the simpler one (inplace). You _may_ have operational needs that make the more complex tag+rev model more appealing, see here for reference, inplace upgrade of everything has these drawbacks, etc etc.

Especially since ztunnel _has_ to be an inplace upgrade anyway no matter which mode you pick, it's not really a choice between "safe" and "unsafe", it's two different models that might be better or worse dependent on your operational setup and needs.

I would rather have an advanced upgrade guide (tags and revisions) under Operations - I don't want to obscure the simplest upgrade instructions, and I don't want to imply everyone should be using tag and rev - most people do not, even today, and it's even less critical that they do for ambient than for sidecars, given the non-requirement for pod restarts.

Reply:
I agree with this. I don't know how we want to express that on the page (tabs, two pages, ...), but it does seem likely we are incorrectly implying (by omission of non-revision docs) you must use revisions to upgrade ambient, which may give the perception ambient is more complex.

Reply:
I mentioned this over in https://github.com/istio/istio.io/pull/15279#discussion_r1643545398

(which also reworks the upgrade docs a bit)

I am fine fixing this in a followup if we want to merge this now, but we simply shouldn't recommend the use of revisions by default/as a matter of first principles for ambient IMO - for install or upgrade, simply because it's a lot of extra work and doesn't actually give you of the benefits you might expect or even need in all cases, if you are coming from sidecars.

We can mention that as a supported install/upgrade topology you _may_ want, depending on your env,  and put it under `/ops`.

Reply:
Mitch, should we make this a separate page?

(We should also think about how we rationalise all these docs when ambient is GA/on by default/etc)